pkg/features/kube_features.go

```diff
@@ -293,6 +293,12 @@ const (
 	// Enables CSI Inline volumes support for pods
 	CSIInlineVolume featuregate.Feature = "CSIInlineVolume"
 
+	// owner: @pohly
+	// alpha: v1.18
```

Comment:
should it be `v1.19` as the v1.18 is already out?

Reply:
Oops! Yes. Fixed.